### DISCUSSION TITLE
图表是否支持点击事件

### DISCUSSION THREAD

**Discussion by GitHublijie1210** [XCharts-Team/XCharts#271] [Q&A]
比如点击柱状图的某个柱子，饼状图的某块扇形区域，有点击事件或接口

**Comment by monitor1394** [comment]
onSerieClick，onSerieEnter和onSerieExit这三个回调。
最好用master的3.8版本，有做了优化，或者等最新release发布。

**Reply by GitHublijie1210** [reply]
感谢回复
